packages/client/components/DiscussionThreadInput.tsx

```diff
@@ -159,8 +159,16 @@ const DiscussionThreadInput = forwardRef((props: Props, ref: any) => {
   }
 
   const ensureHasText = (value: string) => value.trim().length
+  const getCurrentText = () => {
+    const editorEl = editorRef.current
+    if (isAndroid) {
+      if (!editorEl || editorEl.type !== 'textarea') return ''
+      return editorEl.value
+    }
 
-  const hasText = ensureHasText(editorState.getCurrentContent().getPlainText())
+    return editorState.getCurrentContent().getPlainText()
+  }
+  const hasText = ensureHasText(getCurrentText())
```

Comment:
-1 do you mind keeping only these changes in this particular PR? if something isn't working, it's easier to track the change which is causing the issue when PRs are related to only one functionality/issue.

Reply:
@BartoszJarocki These changes only account for the visibility of the button when there is no content in the input for Android devices; according to the issue description, we only pass the first check.

- [X] User can send the comment using the send button
- [ ] Clicking the send button should not close the keyboard
- [ ] Next-line button should change the line (instead of sending a comment) as there is no other way on mobile to change the line.

Does this mean the other acceptance criteria should be implemented in other PRs?

Reply:
@BartoszJarocki These changes only account for the visibility of the button when there is no content in the input for Android devices; according to the issue description, we only pass the first check.

- [X] User can send the comment using the send button
- [ ] Clicking the send button should not close the keyboard
- [ ] Next-line button should change the line (instead of sending a comment) as there is no other way on mobile to change the line.

Does this mean we should implement the other acceptance criteria in other PRs?

Reply:
Good question! There are no strict rules here, but definitely, there's no rule about 1 task = 1 PR. Our code review policy states, `Once you're assigned to a task, it's your responsibility to split it the way you can submit reasonably sized PRs. It’s ok to have multiple PRs for a single issue.`. 

The main reason I ask to split PR is that I don't really understand why for example, part `if (e.key !== 'Enter' || e.shiftKey) return` was removed down below. And while we can discuss that further, the main issue here (send button is disabled on mobile) would be already merged.